src/main/java/net/openhft/chronicle/queue/impl/single/TableStoreWriteLock.java

```diff
@@ -96,7 +96,7 @@ private void lockAssertPostConditions() {
                 ((lockedByThread = Thread.currentThread()) != null && (lockedHere = new StackTrace()) != null);
     }
 
-    private void lockHandleTimeoutException(long currentLockValue) {
+    private void lockHandleTimeout(long currentLockValue) {
```

Comment:
Perhaps add some verb to method name?

Reply:
How about `handleLockTimeout`?

Reply:
It does. I tweaked the naming a little bit more - I also removed the `Exception` substring from another method!